configure.ac

```diff
@@ -102,6 +102,10 @@ AC_CHECK_TOOL(GPERF, gperf)
 AS_IF([test -z "$GPERF"],
   AC_MSG_ERROR([*** gperf not found])
 )
+AC_MSG_CHECKING([whether gperf uses size_t])
+AS_IF([$GPERF --version |grep -q 'gperf 3.[[]1-9[]]'],
```

Comment:
Hmm, should we go with feature detection instead? E.g. this is what systemd did: https://github.com/systemd/systemd/commit/c9f7b4d.